bookstore/__init__.py

```diff
@@ -1,14 +1,14 @@
+# TODO: refactor the import, PACKAGE_DIR, del
```

Comment:
Let's just remove the PACKAGE_DIR from this file. We can just move it to where it's used in `clone.py`

Reply:
I generally prefer to do this at the `__init__.py` so that it can be accessible from other locations, including those outside of the library.

Could you explain the reason you disprefer this pattern?

Reply:
I have removed the TODO to move this forward. We can always refactor at a later time if needed.